integration-tests/src/test/java/oracle/kubernetes/operator/ITMultipleClusters.java

```diff
@@ -0,0 +1,212 @@
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+/**
+ * Multiple clusters in a domain tests
+ *
+ * <p>More than 1 cluster is created in a domain , like configured and dynamic
+ */
+@FixMethodOrder(MethodSorters.NAME_ASCENDING)
+public class ITMultipleClusters extends BaseTest {
+
+  private static Operator operator1, operator2;
+
+  private static Operator operatorForBackwardCompatibility;
+  private static Operator operatorForRESTCertChain;
+
+  private static final String TWO_CONFIGURED_CLUSTER_SCRIPT =
+      "create-domain-two-configured-cluster.py";
+  private static final String TWO_MIXED_CLUSTER_SCRIPT = "create-domain-two-mixed-cluster.py";
+
+  /**
+   * This method gets called only once before any of the test methods are executed. It does the
+   * initialization of the integration test properties defined in OperatorIT.properties and setting
+   * the resultRoot, pvRoot and projectRoot attributes.
+   *
+   * @throws Exception
+   */
+  @BeforeClass
+  public static void staticPrepare() throws Exception {
+    // initialize test properties and create the directories
+    initialize(APP_PROPS_FILE);
+  }
+
+  /**
+   * Releases k8s cluster lease, archives result, pv directories
+   *
+   * @throws Exception
+   */
+  @AfterClass
+  public static void staticUnPrepare() throws Exception {
+    logger.info("+++++++++++++++++++++++++++++++++---------------------------------+");
+    logger.info("BEGIN");
+    logger.info("Run once, release cluster lease");
+    tearDown(new Object() {}.getClass().getEnclosingClass().getSimpleName());
+    logger.info("SUCCESS");
+  }
+
+  /**
+   * Create 2 configured clusters in a domain each having 2 managed servers. Verify the managed
+   * servers are running and verify the basic use cases.
+   *
+   * @throws Exception
+   */
+  @Test
+  public void testCreateDomainTwoConfiguredCluster() throws Exception {
+    String DOMAINUID = "twoconfigclustdomain";
+    String testMethodName = new Object() {}.getClass().getEnclosingMethod().getName();
+    logTestBegin(testMethodName);
+    String template =
+        BaseTest.getProjectRoot() + "/kubernetes/samples/scripts/common/domain-template.yaml";
+    logger.info("Creating Operator & waiting for the script to complete execution");
+    // create operator1
+    if (operator1 == null) {
+      operator1 = TestUtils.createOperator(OPERATOR1_YAML);
+    }
+    Domain domain = null;
+    boolean testCompletedSuccessfully = false;
+    try {
+      Map<String, Object> domainMap = TestUtils.loadYaml(DOMAINONPV_WLST_YAML);
+      domainMap.put("domainUID", DOMAINUID);
+      domainMap.put("clusterType", "CONFIGURED");
+      domainMap.put(
+          "createDomainPyScript",
+          "integration-tests/src/test/resources/domain-home-on-pv/"
+              + TWO_CONFIGURED_CLUSTER_SCRIPT);
+
+      String add =
```

Comment:
create a private method appendCluster2ToDomainTemplate() or some such name and call from both the tests

Reply:
Done